config/staging.yml

```diff
@@ -11,6 +11,7 @@ notify_sms_template_ids:
 
 notify_email_template_ids:
   self_signup_credentials: 96d1f5ac-2ebe-41a7-878f-9a569e0bb55c
+  do_not_reply_credentials: 45d6b6c4-6a36-47df-b34d-256b8c0d1511
```

Comment:
I don't think the name is quite right. Or perhaps it shouldn't be nested. The value above (`self_signup_credentials`) is a template (https://www.notifications.service.gov.uk/services/8a09d848-d453-4aea-bb1e-46f7d19a1814/templates/96d1f5ac-2ebe-41a7-878f-9a569e0bb55c) whereas the new `do_not_reply_credentials` is just an ID for an email address. 

So I don't think it should be nested under `notify_email_template_ids:` because its not a template ID.